doc-utils/server.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import http from "http";
+import express, { Express } from "express";
+import getPort from "get-port";
+import * as ReactDOMServer from "react-dom/server";
+
+export type CreateServerOptions = {
+    scriptSrc: string;
+    styleHref: string;
+    rootId: string;
+    port: number;
+};
+
+const defaults: CreateServerOptions = {
+    scriptSrc: "client.js",
+    styleHref: "client.css",
+    rootId: "app",
+    port: 1234,
+};
+
+export async function createServer(rootComponent: React.ReactElement, options = defaults): Promise<Express> {
+    const opts = { ...defaults, ...options };
+
+    const app = express();
+
+    app.get("/", (req, res) => {
+        const app = ReactDOMServer.renderToString(rootComponent);
+        const html = `<!DOCTYPE html>
+          <html lang="no">
+          <head>
+              <meta charset="utf-8" />
+              <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+              <title>Jøkul DevServer</title>
+              <meta name="viewport" content="width=device-width, initial-scale=1" />
+              <script src="${opts.scriptSrc}" async defer></script>
+              <link href="${opts.styleHref}" rel="stylesheet"></link>
+          </head>
+          <body style="height: 100%; display: flex; align-items: center; justify-content: center">
+              <div class="jkl" id="${opts.rootId}">${app}</div>
+          </body>
```

Comment:
Kan vi gjøre dette? Som det er nå fyller ikke bakgrunnen hele siden, og det er vel strengt tatt ingen grunn til at den ekstra div-en må være der annet enn konvensjon?

```suggestion
          <body class="jkl" id="${opts.rootId}" style="height: 100%; display: flex; align-items: center; justify-content: center">
              ${app}
          </body>
```

Reply:
```
client.js:2331 Warning: createRoot(): Creating roots directly with document.body is discouraged, since its children are often manipulated by third-party scripts and browser extensions. This may lead to subtle reconciliation issues. Try using a container element created for your app.
```